extensions/amp-base-carousel/1.0/README.md

```diff
@@ -0,0 +1,731 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content along a horizontal axis or vertical axis.
+---
+
+<!---
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Bento Carousel
+
+## Usage
+
+A generic carousel for displaying multiple similar pieces of content along a
+horizontal or vertical axis.
+
+Each of the Bento Carousel component’s immediate children is considered an
+item in the carousel. Each of these nodes may also have arbitrary children.
+
+The carousel consists of an arbitrary number of items, as well as optional
+navigational arrows to go forward or backwards a single item.
+
+The carousel advances between items if the user swipes or uses the customizable
+arrow buttons.
+
+This document covers both the web component `<bento-base-carousel>` and the Preact and React functional components `<BentoBaseCarousel>`.
```

Comment:
Maybe replace this with:
"Use the Bento Carousel as a standalone web component `<bento-base-carousel>` or as a Preact and React functional component `<BentoBaseCarousel>`." and link to both sections from here

Reply:
Added this sentence from [this thread](https://github.com/ampproject/amphtml/pull/35645#discussion_r690568800), but added the links recommended here. Thanks.